can/bus.py

```diff
@@ -92,7 +135,7 @@ def send_periodic(self, msg, period, duration=None):
             no duration is provided, the task will continue indefinitely.
 
         :return: A started task instance
-        :rtype: can.CyclicSendTaskABC
+        :rtype: :class:`can.CyclicSendTaskABC`
```

Comment:
Don't think you need to change this. See [Sphinx docs](www.sphinx-doc.org/en/master/domains.html#info-field-lists).